deployments/charts/kuma/values.yaml

```diff
@@ -664,6 +664,15 @@ hooks:
   containerSecurityContext:
     readOnlyRootFilesystem: true
 
+  # -- ebpf-cleanup hook needs write access to the root filesystem to clean ebpf programs
+  ebpfCleanup:
+    # -- Security context at the pod level for crd/webhook/cleanup-ebpf
```

Comment:
Maybe add a comment that this will most likely break everything if either of these two are changed?

Reply:
done